src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
```

Comment:
The constant 'SMALL_DISCOUNT' could be more descriptive. It would be better to name it 'SHORT_TERM_DISCOUNT' to reflect that it applies to shorter rental periods.